src/main/java/uk/gov/hmcts/reform/darts/entities/Annotation.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.hmcts.reform.darts.entities;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import jakarta.persistence.Temporal;
+import jakarta.persistence.TemporalType;
+import jakarta.persistence.Version;
+import lombok.Data;
+
+import java.util.Date;
+
+@Entity
+@Table(name = "moj_annotation")
+@Data
+public class Annotation {
+
+    @Id
+    @Column(name = "moj_ann_id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    @OneToOne
+    @JoinColumn(name = "moj_cas_id")
+    private Case theCase;
+
+    @Column(name = "r_annotation_object_id", length = 16)
+    private String legacyObjectId;
+
+    @Column(name = "c_text", length = 2000)
+    private String text;
+
+    @Temporal(TemporalType.TIMESTAMP)
+    @Column(name = "c_time_stamp")
+    private Date timestamp;
+
+    @Temporal(TemporalType.DATE)
+    @Column(name = "c_start")
+    private Date start;
+
+    @Temporal(TemporalType.DATE)
+    @Column(name = "c_end")
+    private Date end;
+
+    @Column(name = "c_courthouse", length = 64)
+    private String courthouse;
```

Comment:
Do wen really need to put length on these fields in the Java model?

Reply:
ok, guess we create another ticket on backlog to track this and decide how to proceed